.circleci/config.yml

```diff
@@ -126,22 +126,17 @@ jobs:
       - image: "cimg/ruby:3.3.4-node"
     steps:
       - checkout
+      - ruby/install-deps
```

Comment:
Is this doing anything more than just bundle install?

Reply:
This is me getting the admin app consistent with the frontend. Really all of this stuff should be wrapped in an orb command for consistency